service/user-service.js

```diff
@@ -0,0 +1,57 @@
+const UsersModel = require('../models/UsersModel');
+const bcrypt = require('bcrypt');
+const tokenService = require('./token-service');
+const ApiError = require('../api-error')
+
+class UserService {
+
+    constructor() {
+        this.users = UsersModel;
+    }
+
+    async register(email, password){
+        if(this.users.isExists(email)) throw ApiError.BadRequest(`Пользователь с таким почтовым адресом уже существует!`);
+
+        const hashPassword = await bcrypt.hash(password, 8);
+
+        const user = await this.users.add(email, hashPassword);
+
+        const tokens = tokenService.generateToken(user);
+        await tokenService.saveToken(user.email, tokens.refreshToken);
+
+        return {...tokens, user}
+    }
+
+    async login (email, password){
+        if(!this.users.isExists(email)) throw ApiError.BadRequest(`Пользователь не был найден!`);
+
+        const user = this.users.find(email);
+
+        const isPassEquals = await bcrypt.compare(password, user.password);
+        if(!isPassEquals) throw ApiError.BadRequest("Неверный пароль!");
+
+        const tokens = tokenService.generateToken({email});
+
+        await tokenService.saveToken(user.email, tokens.refreshToken);
+        return {...tokens, email}
```

Comment:
You are forgetting `;` from time to time (line 47 as the example). You can install `eslint` as dev dependency for this purpose.